snakemake/notebook.py

```
@@ -66,12 +67,27 @@ def execute_script(self, fname, edit=None):
-                logger.info("Opening notebook for editing.")
+                logger.info(f"Opening notebook for editing at {edit.ip}:{edit.port}")
                 cmd = (
                     "jupyter notebook --browser ':' --no-browser --log-level ERROR --ip {edit.ip} --port {edit.port} "
-                    "--NotebookApp.quit_button=True {{fname:q}}".format(edit=edit)
+                    "--ServerApp.quit_button=True {{fname:q}}".format(edit=edit)
+                )
+            elif has_papermill:
+                if fname_out is None:
+                    output_parameter = fname
+                else:
+                    output_parameter = "{fname_out}"
```

Ruff

Use ternary operator `output_parameter = fname if fname_out is None else "{fname_out}"` instead of `if`-`else`-block

Replace `if`-`else`-block with `output_parameter = fname if fname_out is None else "{fname_out}"`

(SIM108)